src/styles/home.styl

```diff
@@ -156,7 +156,8 @@ a.signin-link {
   top: 50%;
   left: 50%;
   width: 50%;
-  width: 90vh;
+  --diamond-size: 90vh;
+  width: var(--diamond-size);
```

Comment:
Second thought: Could we keep this as `90vh` and animate by changing `scale`? We'll get a lot better performance that way.